drivers/staging/vc04_services/bcm2835-isp/bcm2835_isp_fmts.h

```diff
@@ -0,0 +1,298 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * Broadcom BCM2835 ISP driver
+ *
+ * Copyright © 2019-2020 Raspberry Pi (Trading) Ltd.
+ *
+ * Author: Naushir Patuck (naush@raspberrypi.com)
+ *
+ */
+
+#ifndef BCM2835_ISP_FMTS
+#define BCM2835_ISP_FMTS
+
+#include "vchiq-mmal/mmal-encodings.h"
+
```

Comment:
You've only included the MMAL header, so if this is included before the header defining V4L2_PIX_FMT_* (videodev2.h) then you'll have a compile failure.

Reply:
Done